package.json

```diff
@@ -65,12 +65,16 @@
     "react-cropper": "^2.3.3",
     "react-dom": "^18.2.0",
     "react-icons": "^5.2.1",
+    "react-katex": "^3.0.1",
     "react-live": "^4.1.5",
     "react-pdf": "^7.7.3",
     "react-router-dom": "6.22.3",
     "react-shepherd": "^4.3.0",
     "react-sizeme": "^3.0.2",
     "recharts": "2.12.3",
+    "rehype-katex": "^7.0.0",
```

Comment:
@ThijsZijdel  have we included example tempaltes of how to use these libraries in the the module? Ideally it'd be great if user knows somehow that they can write html markup with katex or math syntax etc.